woocommerce/woocommerce-android
# Issue/9554 disable edit text option coupon removal screen

Closes: #9554 


### Description


This PR:

- Changes the Edit Text Field in the coupon removal screen into a regular text field
- Removes the Validator logic since we are selecting coupons from the coupon selector list now and don't need to validate the coupon code entered by the customer
- Removes the edit coupon logic from the screen above since, again, the user is no longe typing the coupon code to find it. 

### Testing instructions


1. Go to the orders tab
2. add new order
3. add a product
4. click on +Add Coupon
5. Select a coupon from the couponselector list (maybe add more coupons)
6. click on the applied coupons row
7. make sure the text field is not editable
8. make sure the done button is gone
9. check the whole flow to make sure the remove coupon button works

### Images/gif


<img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/2440c4bb-9045-4c45-9464-afbc464c48e3" width="350"/>
